### PR TITLE
FIX: show delete and/or upload only when permitted

### PR DESCRIPTION
**Pull request by realemx**

### CODE REVIEW

File: src/main/java/mops/presentation/DirectoryController.java [hunk 1]
```diff
@@ -42,6 +44,11 @@
      */
     private final FileService fileService;
```
Comment (iTitus) [inline src/main/java/mops/presentation/DirectoryController.java]:
```suggestion
```

File: src/main/resources/templates/directory.html [hunk 2]
```diff
@@ -106,23 +107,24 @@
             <td th:text="${file.owner}">%Owner%</td>
             <td th:text="${file.getSizeString()}">%Größe%</td>
             <td>
-                <form action="#" th:action="@{../file/{fileId}/delete(fileId = ${file.id})}" th:method="post">
+                <form th:if="${deletePermission || file.getOwner() == account.getName()}" action="#" th:action="@{../file/{fileId}/delete(fileId = ${file.id})}" th:method="post">
```
Comment (iTitus) [inline src/main/resources/templates/directory.html]:
```suggestion
                <form th:if="${deletePermission || file.owner == account.name}" action="#" th:action="@{../file/{fileId}/delete(fileId = ${file.id})}" th:method="post">
```

File: src/main/resources/templates/directory.html [hunk 3]
```diff
@@ -106,23 +107,24 @@
...
-    <form action="#" th:action="@{{dirId}/create(dirId = ${directory.getId()})}" th:method="post">
-        <input type="text" name="folderName" required>
-        <input type="submit" name="newFolder" value="Neuer Ordner">
-    </form>
-
-    <form action="#" th:action="@{{dirId}/upload(dirId = ${directory.getId()})}" th:method="post"
-          enctype="multipart/form-data">
-        <input type="file" name="file" required>
-        <input type="submit" name="uploadFile" value="Datei Hochladen">
-    </form>
```
Comment (Segelzwerg) [inline src/main/resources/templates/directory.html]: What happend here?

File: src/main/java/mops/presentation/DirectoryController.java (hunk 1 above)
Comment (realemx) [reply src/main/java/mops/presentation/DirectoryController.java]: Why!?

Comment (realemx) [reply src/main/java/mops/presentation/DirectoryController.java]: nvmd

**Pull request merged**